app/controllers/warehouse_reports_controller.rb

```diff
@@ -2,16 +2,25 @@ class WarehouseReportsController < ApplicationController
   include WarehouseReportAuthorization
   # This page just lists the available reports, each report is responsible for access
   skip_before_action :report_visible?
-  def index
 
+  def index
     all_report_definitions = GrdaWarehouse::WarehouseReports::ReportDefinition.ordered.
       group_by(&:report_group)
     if current_user.can_view_all_reports?
       @report_definitions = all_report_definitions
     else
       @report_definitions = current_user.reports.group_by(&:report_group)
     end
+    @report_definitions = filter_reports(@report_definitions)
   end
 
+  private
```

Comment:
Instead of this, I've added an enabled flag to the report definitions.  If all health reports are disabled, the health tab won't show up.  We'll probably want to build report definition CRUD eventually.